apps/dotcms-ui/src/app/view/components/_common/dot-page-selector/service/dot-page-selector.service.ts

```diff
@@ -124,7 +124,7 @@ export class DotPageSelectorService {
     }
 
     private getSiteName(site: string): string {
-        return site.replace(/\//g, '');
+        return site.replace(/\//g, '').replace(' ', '?');
```

Comment:
Why?

Reply:
that is the way elastic search receive blank spaces.